.github/workflows/build-and-deploy.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       artifact-name: 'ztl-deploy-artifact-v1'
 
   qa-1:
-    uses: ZTL-ARTCC/Current_Website/.github/workflows/deploy.yml@main
+    uses: ZTL-ARTCC/Current_Website/.github/workflows/deploy.yml@cf-purge-cache-deploy
```

Comment:
I don't understand what is going on here. Can you explain this to me? What is @cf-purge-cache-deploy referring to in the deploy script?